watershed_workflow/land_cover_properties.py

```diff
@@ -33,11 +43,33 @@ def compute_time_series(lai, lc):
     df = pd.DataFrame()
     df['time [datetime]'] = lai.times
 
-    for lc in unique_lc:
+    for ilc in unique_lc:
         time_series = [
-            lai.data[i, :, :][np.where(lc.data == lc)].mean() for i in range(len(lai.times))
+            lai.data[itime, :, :][np.where(lc.data[lc_idx, :, :] == ilc)].mean() for itime in range(len(lai.times))
         ]
-        df[f'{lc}'] = time_series
+        col_name = watershed_workflow.sources.manager_modis_appeears.colors[int(ilc)][0]
+        df[f'MODIS {col_name} LAI [-]'] = time_series
+
+    if smooth:
```

Comment:
rather than this, can we ask the user to call utils.interpolate_in_time() and add a new function, utils.smooth() and have them call that (we already have utils.compute_average_year()).  Rather than have every function that generates data accept a "smooth" parameter, I'd rather have the user call two functions, one to get the data and one to smooth/interpolate/whatever it.